src/pages/components/menu/usage.mdx

```diff
@@ -0,0 +1,588 @@
+
+### Anatomy
+
+<Row>
+<Column colLg={12}>
+
+![Menu anatomy diagram](images/usage-menu-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Trigger**: Elements that are triggered to display a list of available
+   actions. These menu triggers can include a menu button, a combo button, and
+   an overflow menu within menu buttons. Menus can also be opened when
+   right-clicking on a page or specific element, like a column, or a row.
+
+2. **Action item**: A menu item that can be activated to apply its action on
+   certain elements. This item can be accompanied by a submenu indicator or
+   keyboard shortcuts.
+
+3. **Divider**: A rule that indicates different sections within the menu.
+
+4. **Submenu indicator**: A caret icon that indicates a submenu.
+
+5. **Keyboard shortcut**: Certain action items may also have keyboard shortcuts.
+
+6. **Menu**: A container displays the open state and contains a list of actions
+   that can be selected.
+
+7. **Selected item**: An item that can be selected. The selection can be single
+   or multi-selected.
+
+8. **Submenu**: A nested level off of the primary level.
+
+### Sizing
+
+#### Height
+
+Menu items come in four sizes: extra small, small, medium, and large. See more
+about size specifications on the
+[style](https://www.carbondesignsystem.com/components/menu/style) tab. When
+using a menu button trigger, the height of the menu item (singular) reflects the
+size of the button being used.
+
+<InlineNotification>
+
+**Note** Currently, Carbon does not support the extra-small size for
+[buttons](https://carbondesignsystem.com/components/button/usage/), only for
+rows or menu items. Therefore, this menu size cannot be utilized with menu
+button triggers. However, the extra-small menu size remains available for
+implementation in other contexts if needed.
+
+</InlineNotification>
+
+| Size        | Height (px / rem) |
+| ----------- | ----------------- |
+| Extra small | 24 / 1.5          |
+| Small       | 32 / 2            |
+| Medium      | 40 / 2.5          |
+| Large       | 48 / 3            |
+
+<Row>
+<Column colLg={8}>
+
+![Menu height](images/usage-menu-height.png)
+
+</Column>
+</Row>
+
+<DoDontRow>
+  <DoDont type="do" caption="Do match the height of the menu buttons and the menu">
+
+![Do match the height of the menu buttons and the menu](images/usage-sizing-height-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not mix the height of the menu buttons and the menu">
+
+![Do not mix the height of the menu buttons and the menu](images/usage-sizing-height-dont.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Width
+
+By default, the [menu button](/components/menu-buttons/usage/#menu-button) and
+[combo button](/components/menu-buttons/usage/#combo-button) follow the
+[button structure style](/components/button/style#structure), and the menu
+maintains a minimum width of 160px to prevent a narrow appearance.
+
+<Row>
+<Column colLg={8}>
+
+![Menu width](images/usage-menu-width-menu-buttons-default.png)
```

Comment:
This image is not showing up
<img width="566" alt="Screenshot 2023-10-30 at 1 25 49 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/fb5b297d-bb79-4aa9-a399-370cd861869c">